src/domain/event/eventInfo/otherEventTimes/OtherEventTimes.tsx

```diff
@@ -51,7 +52,15 @@ const OtherEventTimes: React.FC<Props> = ({
     history.push(eventUrl);
   };
 
-  if (!superEventId || events.length === 0) return null;
+  if (!superEventId) {
+    return null;
+  } else if (events.length === 0 && loading) {
+    return (
+      <div className={styles.skeletonWrapper}>
+        <SkeletonLoader />
+      </div>
+    );
+  }
```

Comment:
After this change we are showing "show other event times" button when events.length === 0

Reply:
Should be `if (!superEventId || events.length === 0) {` ?

Reply:
what is the difference from previous implementation except {}? the whole point of the fix to add loader, Tomi suggested. if we leave previous, then loader will never appear

Reply:
if there is no super id, then null always. if there is superid, then probably there will be other times loaded.. no?

Reply:
It is possible that there are no other events even when superEventId is available. Like in the screenshot

Reply:
Previously it wouldn't show that section at all